src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let calls = 0;
+
+  const getSum = (first) => {
+    calls++;
+
+    const result = (second) => calls <= 3 || calls % 2
```

Comment:
it's hard to read, make it multiline expression, also you can return this function right away